main.css

```diff
@@ -0,0 +1,454 @@
+* {
```

Comment:
- [ ] Kindly make sure to use the provided colors `#EC5242`, `#D3D3D3` in your design